docs/reference/release-notes/7.14.asciidoc

```diff
@@ -3,6 +3,22 @@
 
 Also see <<breaking-changes-7.14,Breaking changes in 7.14>>.
 
+[[known-issues-7.14.2]]
+[discrete]
+=== Known issues
+
+* Aggregations: When a cross cluster search request is proxied, the memory for the aggregations on the
+proxy node will not be freed. The trigger is cross cluster search using aggregations where minimize 
+roundtrips is not effective (e.g. when minimize roundtrips is explicitly disabled, or implicitly disabled 
+when using scroll, async and point in time searches). This issue can happen in all modes of remote connections 
+configured for cross cluster search (sniff and proxy). In sniff mode, we only connect to a subset of the 
+remote nodes (by default 3). So if the remote node we want to send a request to is not one of those 3, 
+we must send the request as a proxy request. The workaround is to periodically restart nodes with heap pressure 
+or cancel the aggregation tasks on the nodes.
```

Comment:
This last bit is not true, there is no way to cancel this task, it's execution has completed but left a leak.

Reply:
good to know thx!